lib/entities/parse_address_from_domain.dart

```diff
@@ -160,6 +161,15 @@ class AddressResolver {
           }
         }
       }
+
+      final thorChainAddress = await ThorChainExchangeProvider.lookupAddressByName(text);
+      if (thorChainAddress != null) {
+        String? address = thorChainAddress[ticker.toUpperCase()];
```

Comment:
the ticker would be `rune` but the returned value from the API will be of chain `THOR` 

https://midgard.ninerealms.com/v2/thorname/lookup/cakewallet

let's add a specific case for this then, since it will probably be just the case for Thor

Reply:
Added case for the 'rune' coin.